### DISCUSSION TITLE
To load an ES module, set "type": "module" in the package.json naive-ui

### DISCUSSION THREAD

**Discussion by jrrmcalcio** [nuxt/nuxt#18264] [Questions]
I tried to use this library: naive-ui (https://www.naiveui.com/en-US/os-theme/docs/usage-sfc)

Just install npm i -D naive-ui vfonts in a fresh installation of nuxt3 and got the error of the ES Modules.  I guess there are many other packages with the same problem, but i wanted to point this one out.

I checked this issue: https://github.com/nuxt/framework/issues/1235 so I create this discussion.

This is the code with error, you can fork it, npm install and npm run dev and when you load the page it crash:

https://github.com/jrrmcalcio/nuxt3-naiveui

**Comment by wobsoriano** [comment]
https://github.com/TuSimple/naive-ui/issues/636

**Comment by jankaderabek** [comment]
I have same problem with vee-validate library. I've already seen issues https://github.com/nuxt/framework/issues/1235 (where @pi0 mentioned that we should create discussion) and https://github.com/logaretm/vee-validate/issues/3550 but I'm not sure what should be next move to solve the problem.

Reproduction for vee-validate: https://codesandbox.io/s/hopeful-nobel-2hxpu

It works fine when I disabled Vite.

Could you please help me to solve this issue? Let me know If I can be helpful with something.
